packages/utils/src/object.js

```diff
@@ -264,3 +264,6 @@ export const camelKeys = object =>
 
 export const snakeKeys = object =>
   Object.fromEntries(Object.entries(object).map(([key, value]) => [snake(key), value]));
+
+export const isNullOrEmptyObject = obj =>
+  obj === null || obj === undefined || !(typeof obj === 'object' && Object.keys(obj).length > 0);
```

Comment:
Nice tests! They picked this up. 
```suggestion
  obj === null || obj === undefined || (typeof obj === 'object' && !Object.keys(obj).length > 0);
```

Reply:
Ah woops!! Hooray for tests 👍

Reply:
Btw, curious to know if you think checking for `undefined` is correct here? I suppose it could say `isNullOrUndefinedOrEmptyObject` but that just feels long winded to me 😛

Reply:
I haven't actually got back to reviewing this, but seeing as you ask I'd go with `isNullishOrEmptyObject`

Reply:
Nullish seems pretty accepted as "null or undefined", see e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator